app/controllers/users_controller.rb

```diff
@@ -7,6 +13,11 @@ def show
     redirect_to root_path
   end
 
+  def index
+    @pagy, @users = pagy User.order(:name), items:
+    Settings.page_10
```

Comment:
```suggestion
      Settings.page_10
```
ngoài ra, chỗ `User.order(:name)` thì chị tha , chư schisnh xác là viết scope trong model User ấy 
```
scope :order_by_name, ->{order :name}
```
ở đây chỉ cần `User.order_by_name`

Reply:
oke ạ, cái này em cũng chưa thấy bao h nãy em tra thì nó bảo dùng order như kia ạ